src/managers/input/InputManager.cpp

```diff
@@ -906,6 +908,11 @@ void CInputManager::applyConfigToKeyboard(SKeyboard* pKeyboard) {
         KEYMAP = xkb_keymap_new_from_names(CONTEXT, &rules, XKB_KEYMAP_COMPILE_NO_FLAGS);
     }
 
+    if (pKeyboard->xkbTranslationState) {
+        xkb_state_unref(pKeyboard->xkbTranslationState);
```

Comment:
1: style, no {} around short ifs
2: leak: not destroyed on .destroy

Reply:
Of course I would check to make sure the ref counting was correct for the keymap and then forget about a destructor 🤦

I checked and [xkb_state_unref](https://github.com/xkbcommon/libxkbcommon/blob/58d0bdc743f3b63cfc305ec06ad736072d9b7008/src/state.c#L609) does its own null check, so I just removed the if. FYI clang-format did not remove those braces for me. There's a `RemoveBracesLLVM` rule that might give you the behavior you want.

Thanks again for the review!

Reply:
RemoveBracesLLVM IIRC removes too much. We only remove around short ifs.

Reply:
RemoveBracesLLVM does indeed also remove braces around single statement for and while, maybe others. Just to make sure I'm understandig you right for my next PR: you keep braces around the other control-flow statements?